its/ruling/src/test/resources/expected/python-S930.json

```diff
@@ -1,40 +1,69 @@
 {
```

Comment:
As a general habit, I think it would be good to leave a small explanation about ruling changes in the PR description, especially when it's about accepted FPs.
If I understand correctly, all changes we're seeing here are due to projects using the outdated PyYAML 5.4 version?

Reply:
Oh yes sorry. Exactly so those are "FPs" as we are analysing the outdated version of PyYAML. I have created a ticket as well to update these sources. Because for example salt, made the change to the new version of PyYAML but we are still analysing an old version of salt that uses PyYAML 5.4